src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java

```diff
@@ -0,0 +1,395 @@
+package pl.edu.pw.mini.msi.knowledgerepresentation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.ActionDomain;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentenceParts.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentences.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.grammar.ActionLanguageBaseListener;
+import pl.edu.pw.mini.msi.knowledgerepresentation.grammar.ActionLanguageParser;
+
+public class ActionLanguageListener extends ActionLanguageBaseListener {
+
+    private static final Logger log = LoggerFactory.getLogger(ActionLanguageListener.class);
+
+    private Action lastAction = null;
+    private Action preLastAction = null;

```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2\.